### ISSUE TITLE
Add bibtex or reference for citation purposes

### ISSUE THREAD

**Issue by hsgg** [ziotom78/Healpix.jl#65]
Hi, do you have a paper, bibtex file, arXiv link, or similar that I can use to cite Healpix.jl? I think that would be very useful for making it easy to properly credit the work you are doing.

**Comment by hsgg** [comment]
Both JOSS and ASCL would certainly satisfy my desire (they are both in ads). However, I am not very familiar with either.

In my estimation the JOSS submission may indeed be out of scope. Would probably require something like running on GPUs or so. (But I think that might be more for libsharp?)

**Comment by ziotom78** [comment]
> Would probably require something like running on GPUs or so. (But I think that might be more for libsharp?)

I agree. Consider that Healpix 3.0 has the potential to keep maps on GPUs, thanks to @xzackli's pull request #26, but so far nobody has reported noteworthy applications or results.

I'll go with the ASCL route; this does not prevent us submitting a paper to JOSS later, but at least we fix this issue quickly.

**Comment by lbenet** [comment]
Is there any progress in the ASCL submission?

**Comment by ziotom78** [comment]
Just yesterday I received an email from Alice Allen saying that she registered our package, its code is `ascl:2109.028` (see the [webpage](http://ascl.net/code/v/3025)). The reason why it took so long is that they could not find any reference to it in any published paper; at the end she was convinced because of the existence of this very GitHub issue.

I already changed the README with proper citing information, but I have not pushed it because I see that the ADS link is not working yet; the link is important because it provides the actual BibTeX entry to use in citations. However, I realized that for the moment I can create a PR and guessing up how the final BibTeX will look like; will merge it when ADS gets updated.

**Comment by ziotom78** [comment]
Here is the link to the new section in the README: [Citing Healpix.jl](https://github.com/ziotom78/Healpix.jl/blob/cff2e8c7dda464b818bc7485886c4ab15a2a647d/README.md#citing-healpixjl), which will appear in the front page once PR #66 is merged.

**Comment by lbenet** [comment]
Thanks a lot!
